docs/README-DEPLOYMENT.md

```diff
@@ -1,54 +1,63 @@
-# Deployment
+# Docs deployment guidelines
 
-Our server configurations watch changes for `latest` tag for an image. Refresh itself If detect newer version.
+This page covers guidelines for deploying the [Handsontable docs](https://handsontable.com/docs).
 
-## From the console
+## About docs deployment
 
-**Once:**
+A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) with the largest version number gets automatically tagged as the docs' `:latest` version.
 
-Login into ghcr:
- * Login: email used for GH Account
- * Pass: PAT with `write:packages` permission: https://github.com/settings/tokens/new
+Our server configuration watches for images tagged as [`:latest`](./README-EDITING.md#editing-the-latest-docs-version), and automatically refreshes after detecting a newer version.
 
-```shell script
-docker login --registry docker.pkg.github.com
-```
+## Docs deployment
 
-**Deploy:**
+To deploy the docs from the console:
 
-```shell script
-docker build -t docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest .
-docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
-```
+1. When deploying for the first time, log in to the GitHub Container Registry (ghcr.io):
+    ```bash
+    docker login --registry docker.pkg.github.com
+    ```
+    * Login: Your GitHub account email
+    * Password: PAT with the `write:packages` permission: https://github.com/settings/tokens/new
 
-## From GH Action
+2. Deploy the docs:
+    ```bash
+    npm run docs:docker:build
+    # npm run docs:docker:build:production # Production build
 
-It happens automatically for each commit pushed into the `develop` branch.
+    docker push docker.pkg.github.com/handsontable/handsontable/handsontable-documentation:latest
+    ```
 
-## Manually from GH Action
+### Deploying the docs from GitHub Actions
 
-It is able to run manually deployment from any branch. To perform a deployment manually:
+GitHub Actions deploys the docs automatically for each commit pushed to the `develop` branch.
```

Comment:
docs > documentation

Reply:
for each commit > not sure about this, but maybe "after each commit" would be a better fit here?